adafruit_pca9685.py

```diff
@@ -122,7 +122,9 @@ class PCA9685:
     def __init__(self, i2c_bus, *, address=0x40, reference_clock_speed=25000000):
         self.i2c_device = i2c_device.I2CDevice(i2c_bus, address)
         self.channels = PCAChannels(self)
+        """Sequence of 16 `PWMChannel` objects. One for each channel."""
```

Comment:
I think you mean to use comments here, not strings?

Reply:
Nope, this is how sphinx (and I think Python) do attribute doc strings. They appear in RTD as expected: https://circuitpython.readthedocs.io/projects/pca9685/en/latest/api.html#adafruit_pca9685.PCA9685.channels